treeherder/webapp/api/performance_data.py

```diff
@@ -757,8 +757,10 @@ def list(self, request):
         if not query_params.is_valid():
             return Response(data=query_params.errors, status=HTTP_400_BAD_REQUEST)
 
-        base_revision = query_params.validated_data['base_revision']
-        new_revision = query_params.validated_data['new_revision']
+        startday = query_params.validated_data['startday']
```

Comment:
`startday` and `endday` are used to retrieve performance datum when no revision is provided for the base (comparing without a base).